azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/storage/AzureBlobStorage.java

```diff
@@ -93,17 +81,4 @@ private BlobContainerClient constructBlockBlobClient(
       throw (e);
     }
   }
-
-  @Override
-  public void deleteBlob(String blobFile, String blobContainerName, String authToken) {
```

Comment:
@mspector this was unused so I've removed it. If that's a bad call... let me know!